tests/unit_tests/c_wrappers/res/enkf/test_ert_config.py

```diff
@@ -186,10 +190,6 @@ def test_extensive_config(setup_case):
         snake_oil_structure_config["FORWARD_MODEL"]
         == ert_config.forward_model_job_name_list()
     )
-    assert (
-        HistorySource[snake_oil_structure_config["HISTORY_SOURCE"]]
-        == model_config.history_source
-    )
```

Comment:
Just out of curiosity, this was removed on purpose? Maybe not applicable anymore?

Reply:
Yes, removed on purpose as we test that in other ways.